### DISCUSSION TITLE
ENG:Plugins and mods do not work through their own bat, but mods work through run.bat RU: плагины и моды не работают через свой bat, а моды работают через run.bat

### DISCUSSION THREAD

**Discussion by Pro100Plus** [IzzelAliz/Arclight#1183] [Q&A]
[crash-2024-01-06_22.40.25-server.txt](https://github.com/IzzelAliz/Arclight/files/13851083/crash-2024-01-06_22.40.25-server.txt)
RU:Запускаю через свой bat(код: java -jar arclight-forge-1.19.2-1.0.3.jar nogui) файл и сервер стартует но потом происходит crash, по логам видно что он видит плагины и вроде с модами тоже ок но он все равно выключает сервер. Через run.bat работают только моды. Напрямую через jar тоже запускал
ENG(Google translate):I launch the file through my bat (code: java -jar arclight-forge-1.19.2-1.0.3.jar nogui) and the server starts but then a crash occurs, the logs show that it sees plugins and it seems that the mods are also ok, but it’s fine also turns off the server. Only mods work through run.bat. I also launched directly through jar

**Comment by TTTOMZZZ** [comment]
I seem to have the same problem with you, initially I start my server with the arclight.jar, and I see a run.bat generated. My plugins does no work with the generated run.bat. So I tried to create a start.bat with (java -Xmx4112M -Xms1028M -jar arclight.jar nogui), and it worked!
